chat-plugins/animals.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const http = require('http');
+
+exports.commands = {
+    animal: 'animals',
+    animals: function(target, room, user) {
+        let tarId = toId(target)
+        let validTargets = {
+            'cat': 'cat',
+            'otter': 'otter',
+            'dog': 'dog',
+            'bunny': 'bunny',
+            'pokemon': 'pokemon',
+            'kitten': 'kitten',
+            'puppy': 'puppy'
+            
+        };
+        let validTarget = validTargets[tarId];
+        if (room.id === 'lobby' && validTarget) return this.errorReply("This command cannot be broadcasted in the Lobby.");
+        if (!validTarget || tarId === 'help') return this.parse('/help animals');
+        let self = this;
+        let reqOpt = {
+            hostname: 'api.giphy.com', // Do not change this
+            path: '/v1/gifs/random?api_key=dc6zaTOxFJmzC&tag=' + tarId,
+            method: 'GET',
+            
+        };
+        let req = http.request(reqOpt, function(res) {
+            res.on('data', function(chunk) {
+                try {
+                    let data = JSON.parse(chunk);
+                    let output = '<center><img src="' + data.data["image_url"] + '" width="400"></center>';
+                    if (!self.runBroadcast()) return;
+                    if (data.data["image_url"] === undefined) {
+                        self.errorReply("ERROR: Command has crashed [No Images found]");
+                        return room.update();
+                        
+                    } else {
+                        self.sendReplyBox(output);
+                        return room.update();
+                        }
```

Comment:
I honestly don't see it; I might be blind but I was using the indent that git gave me lma0